apps/shooting-clubs/acceptance/features/location-address.js

```diff
@@ -0,0 +1,220 @@
+'use strict';
+
+const steps = require('../../index');
+
+Feature('Shooting club - location address');
+
+Before((
+  I,
+  locationAddressPage
+) => {
+  I.visitPage(locationAddressPage, steps)
+});
+
+// Location postcode step
```

Comment:
You can remove these comments as you are specifying the step in you scenario description